src/Process/FeatureProcessing.js

```diff
@@ -20,15 +19,12 @@ export default {
         if (node.layerUpdateState[layer.id] === undefined) {
             node.layerUpdateState[layer.id] = new LayerUpdateState();
         } else if (!node.layerUpdateState[layer.id].canTryUpdate()) {
+            // toggle visibility features
+            const features = node.link.filter(n => n.layer?.id == layer.id);
```

Comment:
It is a detail, but it might be more relevant to rename `n` parameter to `feature` or `f`, since `node.link` contains `FeatureMeshes`.